### DISCUSSION TITLE
Italian translation

### DISCUSSION THREAD

**Discussion by robertotravagliante** [lllyasviel/Fooocus#1931] [Q&A]
I want to make the Italian translation for you. I already started to modify the json. How can I send my work, when it will be completed?

**Comment by poor7** [comment]
https://github.com/lllyasviel/Fooocus/pull/1817

**Comment by mashb1t** [comment]
@robertotravagliante thank you for the offer but the italian translation already exists as pointed out. Feel free to download the file and check if this works for you.

**Reply by robertotravagliante** [reply]
ok, well done, have a great day
